src/server/routes/root.js

```diff
@@ -14,8 +14,8 @@ const htmlDirectory = path.join(__dirname, "../../../dist/client/views");
  * @param {boolean} [isError=false] - If the file is an error page.
```

Comment:
Can you update this parameter to be about the localized argument? Sorry I realized I have it describing an old unused argument.